concepts/sbtc/auxiliary-features/signer-wallet-rotation.md

```diff
@@ -0,0 +1,41 @@
+# Signer Wallet Rotation
+
+Signer Wallet Rotation is a crucial security feature in the sBTC system that allows sBTC Signers to rotate their private keys securely.
+
+## Overview
+
+- sBTC Signers have the ability to rotate their private keys.
+- This feature enhances the long-term security of the sBTC system.
+- Key rotation is coordinated among signers and requires on-chain voting.
```

Comment:
this is  a bit unclear - does a key rotation need a vote from the full community, or is it a smaller vote (i.e. the signers themselves call a vote with the other signers)

Reply:
@AshtonStephens any additional context you can give here?